lib/Mml-lab.js

```diff
@@ -55,7 +55,7 @@ const Lab = window.Lab = {
 }
 
 if (window.location.search !== "") {
-  Lab.mml.value = decodeURIComponent(window.location.search.substr(1));
+  Lab.mml.value = decodeURIComponent(window.location.search.substr(1)).trim();
```

Comment:
Should a similar change be made in the `TeX-lab.js` file?

Reply:
No, that's not necessary as the texparser deals with whitespace surrounding an expression.
On the other hand, we could possibly move the whitespace trimming into the mathml.compile method.